index.html

```diff
@@ -289,6 +289,16 @@ <h2>About me</h2>
           </ul>
         </div>
       </div>
+
+      <section id="contact-me">
+        <h2>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+        <form>
+            <input class="contact-me-field" type="text" name="name" maxlength="30" placeholder="Full name" required>
+            <input class="contact-me-field" type="email" name="email" placeholder="Email address" required>
+            <textarea id="contact-me-message" name="message" maxlength="500" placeholder="Enter text here" required></textarea>
+            <button type="submit">Get in touch</button>
+        </form>
+      </section>
```

Comment:
As mentioned by the previous reviewer, please make sure to follow the contact form design provided by the [template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471) you chose, as this one is still missing the social icons section after the contact form :eyes: :exclamation: 

Your contact form:
![image](https://user-images.githubusercontent.com/92395484/190704871-7e5cca5c-f289-40da-9743-7f8ffa3a52b8.png)

The contact form from the template:
![image](https://user-images.githubusercontent.com/92395484/190705465-c731eba6-3143-4d41-a2da-a88f7c833726.png)

**[NOTE]:** I would highly recommend sticking to design as much as possible (fonts, margins, etc) :100: :art: